src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -393,6 +496,11 @@ public IA11yElement GetFirstChild()
             return Children[0];
         }
 
+        /// <summary>
+        /// Finds the first decendent element matching the given condition.
+        /// </summary>
+        /// <param name="condition">a function that returns true if the given descendant element meets a set of criteria</param>
+        /// <returns>an <see cref="IA11yElement"/> object representing the matching descendant if one exists; otherwise, false</returns>
         public A11yElement FindDescendent(Func<A11yElement, Boolean> condition)
```

Comment:
Spelling: Should end in "ant", not "ent"

```suggestion
        public A11yElement FindDescendant(Func<A11yElement, Boolean> condition)
```